assets/javascripts/wizard/components/validator.js.es6

```diff
@@ -0,0 +1,43 @@
+import Component from "@ember/component";
+import { not } from "@ember/object/computed";
+import { ajax } from "discourse/lib/ajax";
+import { getToken } from "wizard/lib/ajax";
+
+export default Component.extend({
+    classNameBindings: ['isValid', 'isInvalid'],
+    validMessageKey: null,
+    invalidMessageKey: null,
+    isValid: null,
+    isInvalid: not('isValid'),
+    layoutName: 'components/validator', // useful for sharing the template with extending components
+    init() {
+      this._super(...arguments);
+
+      if (this.get('validation.backend')) {
+        // set a function that can be called as often as it need to
+        // from the derived component
+        this.backendValidate = (params) => {
+          return ajax('/realtime_validations', {
```

Comment:
@fzngagan Minor point, but the discourse convention is to have dasherized urls, not underscored urls. This should be ``/realtime-validations``.